tests/model_inheritance_regress/tests.py

```diff
@@ -182,15 +182,6 @@ def test_issue_6755(self):
         self.assertEqual(r.id, orig_id)
         self.assertEqual(r.id, r.place_ptr_id)
 
-    def test_issue_7488(self):
-        # Regression test for #7488. This looks a little crazy, but it's the
-        # equivalent of what the admin interface has to do for the edit-inline
-        # case.
-        suppliers = Supplier.objects.filter(
-            restaurant=Restaurant(name='xx', address='yy'))
-        suppliers = list(suppliers)
-        self.assertEqual(suppliers, [])
-
```

Comment:
The test is moved to tests/queries/tests.py `Ticket31486Test`, renamed and changed. For more details look at comment https://code.djangoproject.com/ticket/31486#comment:8, please